django-backend/fecfiler/web_services/tasks.py

```diff
@@ -0,0 +1,48 @@
+from celery import shared_task
+from fecfiler.f3x_summaries.models import F3XSummary
+from fecfile_validate import validate
+from django.db import models
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+F3X_SCHEMA = validate.get_schema("F3X")
+F3X_PROPERTIES = F3X_SCHEMA.get("properties", {}).items()
+F3X_COLUMNS = {v.get("fec_spec", {}).get("COL_SEQ", None): k for k, v in F3X_PROPERTIES}
+F3X_NUMBER_OF_COLUMNS = max(F3X_COLUMNS.keys())
+SERIALIZERS = {
+    models.BooleanField: lambda b, o, fn: "X" if b else "",
+    models.DateField: lambda d, o, fn: d.strftime("%Y%m%d") if d else "",
+    models.ForeignKey: lambda fo, o, fn: getattr(o, fn + "_id", ""),
+    None: lambda v, o, fn: str(v) if v else "",
+}
```

Comment:
Is there anything that can be done to make this section a little more readable?

Reply:
E.G. `models.ForeignKey: lambda fo, o, fn: getattr(o, fn + "_id", "")`

I _think_ that "fo" is the foreign key, and "o" is the resulting object, but I would not be able to tell you why it's trying to get `fn + "_id"` from "o"

Reply:
great suggestion!  I left my code golf hat on while writing that.  updating now